### PR TITLE
Call to the api implemented

### PR DESCRIPTION
**Pull request by juanpablo-pereira**
### Repository calling to the api requested by the MoviesBloc

Movies - Debugguer
Movies - Postman

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -1 +1,16 @@
...
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      debugShowCheckedModeBanner: false,
+      title: 'Movies App',
+      home: MoviesPage(title: 'Movies list'),
```
Comment (FranciscoGaraventa) [inline lib/main.dart]: Missed a trailing comma here

File: lib/src/bloc/movies_bloc.dart [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import 'dart:async';
+
+import '../model/movies_model.dart';
+import '../resources/movies_repository.dart';
+import 'IMoviesFetcherBloc.dart';
```
Comment (MarcosCGlobant) [inline lib/src/bloc/movies_bloc.dart]: keep the same convention for the name of the files, lower case separated with low hyphens, for example: i_movies_fetcher_bloc.

But in this case the interface should have the same name as the class, less the "i", you should rename it to IMoviesBloc and the file i_movies_bloc, to keep consistency between files

File: lib/src/bloc/movies_bloc.dart [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+class MoviesBloc extends IMoviesFetcherBloc {
+  MoviesRepository moviesRepository = MoviesRepository();
+  StreamController<MoviesModel> moviesStreamController = StreamController();
```
Comment (FranciscoGaraventa) [inline lib/src/bloc/movies_bloc.dart]: Both variables are private inside bloc, add '_' at beginning

File: lib/src/model/movies_result_model.dart [hunk 4]
```diff
@@ -0,0 +1,65 @@
+class Result {
```
Comment (MarcosCGlobant) [inline lib/src/model/movies_result_model.dart]: the class should have the same name as the file, to keep consistency

**Review by MarcosCGlobant — COMMENTED** [review]
Minor files naming comments

File: lib/src/model/movies_result_model.dart (hunk 4 above)
Comment (FranciscoGaraventa) [inline lib/src/model/movies_result_model.dart]: Use a more descriptive class name

File: lib/main.dart (hunk 1 above)
Comment (juanpablo-pereira) [reply lib/main.dart]: Done

**Review by MarcosCGlobant — APPROVED** [review]
LGTM

File: lib/src/bloc/movies_bloc.dart (hunk 3 above)
Comment (juanpablo-pereira) [reply lib/src/bloc/movies_bloc.dart]: I have the doubt if that was necessary, great, thanks, Added

File: lib/src/bloc/movies_bloc.dart (hunk 2 above)
Comment (juanpablo-pereira) [reply lib/src/bloc/movies_bloc.dart]: Changed

File: lib/src/model/movies_result_model.dart (hunk 4 above)
Comment (juanpablo-pereira) [reply lib/src/model/movies_result_model.dart]: Done

Comment (juanpablo-pereira) [reply lib/src/model/movies_result_model.dart]: Done

**Review by FranciscoGaraventa — APPROVED** [review]
LGTM

**Pull request merged**